### PR TITLE
developers.google.com, mojosoft.com.pl thecamels.org

### PR DESCRIPTION
**Pull request by FadeMind**
SITE 
https://developers.google.com/speed/public-dns/docs/troubleshooting

BEFORE

AFTER

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2693,6 +2693,9 @@ a.button,
 .devsite-landing-row-item {
     border-color: var(--darkreader-border--devsite-select-border) !important;
 }
+.pre-style, code, pre {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2696]:
```suggestion
.pre-style,
code,
pre {
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -6665,15 +6668,24 @@ body:not(.heimu_toggle_on) .new:hover .heimu.off {
...
+INVERT
+img[src*="logo16"]
+img[src*="img/iko"]
+img[src*="img/ideabank"]
+
 CSS
 #viewsh,
 #view,
+header,
 header.smaller {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6680]: This selector is now redundant due to `header`

**Review by Gusted — APPROVED** [review]

**Comment by FadeMind** [comment]
https://thecamels.org/

**Pull request merged**